app/controllers/chat_controller.rb

```diff
@@ -452,6 +452,7 @@ def preloaded_chat_message_query
       .includes(in_reply_to: [:user, chat_webhook_event: [:incoming_chat_webhook]])
       .includes(:revisions)
       .includes(:user)
+      .includes(user: :user_status )
```

Comment:
Should we do this only when `SiteSetting.enable_user_status` is enabled?

Reply:
We should! Fixed.